src/control/SConscript

```diff
@@ -236,18 +235,29 @@ def scons():
         return
 
     senv = denv.Clone()
-    denv = senv
-    prereqs.require(denv, 'spdk')
 
     denv.AppendENVPath("CGO_CFLAGS", denv.subst("$_CPPINCFLAGS"), sep=" ")
 
     SConscript('lib/spdk/SConscript', exports='denv')
 
-    denv.AppendUnique(LINKFLAGS=["-Wl,--no-as-needed"])
-    prereqs.require(senv, 'pmdk', 'spdk', 'ofi', 'hwloc')
+    prereqs.require(senv, 'ofi', 'hwloc')
     daos_build.add_rpaths(denv, "..", True, True)
 
-    cgolibdirs = senv.subst("-L$BUILD_DIR/src/control/lib/spdk "
+    # Copy setup_spdk.sh script to be executed at daos_server runtime.
+    senv.Install(join('$PREFIX', 'share/daos/control'),
+                 join(gosrc, 'server/init/setup_spdk.sh'))
+
+    serverbin = install_go_bin(senv, gosrc, None, "daos_server",
+                               "daos_server")
+
+    aenv = denv.Clone()
+    denv = aenv
+    prereqs.require(denv, 'spdk')
+
+    aenv.AppendUnique(LINKFLAGS=["-Wl,--no-as-needed"])
+    prereqs.require(aenv, 'pmdk', 'spdk', 'ofi', 'hwloc')
+    aenv.Replace(RPATH=[])
+    cgolibdirs = aenv.subst("-L$BUILD_DIR/src/control/lib/spdk "
```

Comment:
The cart and gurt stuff isn't needed by server?

Reply:
It seems to build OK without it...